tools/operator/scripts/migrateToRaft.sh

```diff
@@ -70,29 +69,71 @@ migrate(){
   echo '{"payload":{"header":{"channel_header":{"channel_id":"'$CHANNEL_NAME'", "type":2}},"data":{"config_update":'$(cat modified_update.json)'}}}' | jq . > modified_update_in_envelope.json
   configtxlator proto_encode --input modified_update_in_envelope.json --type common.Envelope --output modified_update_in_envelope.pb
   peer channel update -f modified_update_in_envelope.pb -c $CHANNEL_NAME -o $CORE_PEER_ADDRESS --tls --cafile $CORE_PEER_TLS_ROOTCERT_FILE --ordererTLSHostnameOverride $ORDERER_NAME
-   rm *.json *.pb
+  rm *.json *.pb
+  cd $CURRENT_DIR
+}
```

Comment:
In line number 58 we are cd into the config directory. As we are calling the other script which is in the scripts directory, I am doing in cd at here

Reply:
line 58 is not inside function migrate(). You call migrate() from lines 79 and 82, in global file scope.

Reply:
I am cd into the config directory to pull the latest config block, converting it into config.json file, copying that into modified_cnfig.json, again converting into .pb files and submitting the config update. Instead of creating all these in the root folder, I am just creating a config folder and using it

Reply:
Got it Scott